ext/ffi/ffi.stub.php

```diff
@@ -11,11 +11,11 @@ final class FFI
 
         public static function cdef(string $code = "", ?string $lib = null): FFI {}
 
-        public static function load(string $filename): ?FFI {}
+        public static function load(string $filename): FFI {}
```

Comment:
I think it can return NULL during preloading?

Reply:
You are right. Thanks for catching this.
Everything else is OK?